flask_iiif/cache/filesystem.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Flask-IIIF
+# Copyright (C) 2014, 2015 CERN.
+#
+# Flask-IIIF is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Implement a simple cache."""
+import os
+
+from werkzeug.contrib.cache import FileSystemCache
+
+from flask import current_app
+
+from .cache import ImageCache
+
+
+class ImageFileSystemCache(ImageCache):
+
+    """Filesystem image cache."""
+
+    def __init__(self):
+        """Initialize the cache."""
+        super(ImageFileSystemCache, self).__init__()
+        cache_dir = os.path.join(current_app.config['CFG_TMPDIR'], 'iiif')
+
+        # Create cache folder if not exist
```

Comment:
This is because Werkzeugs `FileSystemCache` requires its own folder (http://werkzeug.pocoo.org/docs/0.10/contrib/cache/)